deployments/kubernetes/chart/ingressmonitorcontroller/values.yaml

```diff
@@ -6,6 +6,7 @@ kubernetes:
 ingressMonitorController:
```

Comment:
All the changes made in this file should reflect in values.yaml.tmpl since that template overrides the chart's values file

Reply:
changed. review plz